packages/batcher/batcher-transaction-poster/src/index.ts

```diff
@@ -95,16 +96,18 @@ class BatchedTransactionPoster {
       value: '0x' + Number(this.fee).toString(16),
       gasLimit: estimateGasLimit(msg.length),
     });
-    const receipt = (await transaction.extra.wait())!;
+    // TODO ONLY ACTIVATE IN ASYNC MODE!!!1!
```

Comment:
This is something I'm still looking into. Waiting for confirmation adds a couple of seconds.

Reply:
I've set up an ENV variable for this.
There looks like no great solution, but it can take 3-4 seconds to get the confirmations.
When using the paima-events you want to know the input-hash quickly, and expect to be posted eventually.